examples/nlp_finetuning/start_with_hpo.py

```diff
@@ -0,0 +1,53 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
do we want to have each example with both hpo and without? seems a bit too much for me

Reply:
Yes, I'm happy to have this one without HPO only.